pack/return/rttcu_encounter.json

```diff
@@ -20,5 +539,569 @@
+        "quantity": 1,
+        "shroud": 2,
+        "text": "<b>Forced</b> - After the Lounge is revealed: Put the set-aside Vault and Library locations into play.\n[action] Investigators at the Lounge spend 1 [per_investigator] clues, as a group, Put the set-aside Hudden Passageway location into play.",
+        "traits": "Lodge.",
+        "type_code": "location"
+    },
+    {
+        "back_name": "Hidden Passageway",
+        "back_text" : "<b>Forced</b> - When you enter Hidden Passageway: Test [agility] (3). If you fail, place 1 doom on the nearest [[Cultist]] enemy as they hear you shifting the bookcase.",
+        "back_illustrator": "Yoann Boissonnet",
+        "clues": 1,
+        "code": "54044",
+        "double_sided": true,
+        "encounter_code": "return_to_for_the_greater_good",
+        "encounter_position": 3,
+        "faction_code": "mythos",
+        "illustrator": "Tiziano Baracchi",
+        "name": "Relic Storage",
+        "pack_code": "rttcu",
+        "position": 44,
+        "quantity": 1,
+        "shroud": 2,
+        "text": "While an investigator controls the [skull] key, Relic storage is connected to Lodge Catacombs, and vice versa.\n<b>Forced</b> - After Relic Storage is revealed: Place 1 random set-aside key on it.",
+        "traits": "Lodge.",
+        "type_code": "location"
+    },
+
+    {
+        "back_name": "Sanctum Doorway",
+        "back_flavor" : "Beyond a stone archway lies an unmarked iron door, mocking you with untold secrets and hidden truths.",
+        "back_illustrator": "Yoann Boissonnet",
+        "clues": 0,
+        "code": "54045",
+        "double_sided": true,
+        "encounter_code": "return_to_for_the_greater_good",
+        "encounter_position": 4,
+        "faction_code": "mythos",
+        "illustrator": "Henning Ludvigsen",
+        "name": "Shrouded Archive",
+        "pack_code": "rttcu",
+        "position": 45,
+        "quantity": 1,
+        "shroud": 4,
+        "text": "[action] - Choose a ([skull]/[cultist]/[tablet]/[elder_thing]) key on a card and test [willpower] or [intellect] (8). Reduce the difficulty of this test by 1 for each [[Silver Twilight]] enemy in play. If you succeed, swap the chosen key with another key in play or with a set-aside ([skull]/[cultist]/[tablet]/[elder_thing]) key.",
+        "traits": "Lodge. Sanctum.",
+        "type_code": "location"
+    },
+    {
+        "back_text": "- When choosing which version of Acts 3 and 4 to use during setup, add the following bullet point above all the others:\n- <i>If the investigator sided with the Lodge</i> and <i>Erynn joined the investigators</i>, use the new Act 3 - \"Beyond the Mist (v. V)\" and act 4 - \"The Binding Rite.\"<hr/>\nFor the duration of this scenario, the following additional rule applies:\n\"While resolving The Watcher's Grasp treachery, The Spectral Watcher does not exhaust after it attacks.\"\nAttach this card to the scenario reference card as a reminder.\",",
```

Comment:
Please cap a for 'Act 4 - The Binding Rite'. 

No / needed in hr tag.

Reply:
At the end of the line, please delete the extra punctuation (\",) so that it reads:

`...as a reminder.",`